VoodooPS2Controller/ApplePS2Device.h

```diff
@@ -27,11 +27,12 @@
 #include <kern/queue.h>
 #include <IOKit/IOService.h>
 #include <IOKit/IOLib.h>
+#include <IOKit/IOInterruptEventSource.h>
 #include <architecture/i386/pio.h>
 
 #ifdef DEBUG_MSG
 #define DEBUG_LOG(args...)  do { IOLog(args); } while (0)
-#define INFO_LOG(args...)  do { IOLog(args); IOSleep(1000); } while (0)
+#define INFO_LOG(args...)  do { IOLog(args); } while (0)
```

Comment:
This delay was done to avoid missed messages during debugging from what I remember. Did you get a kernel panic with it? If so, maybe we should separate logs where we can delay and where we cannot (i.e. interrupt delays).

cc @usr-sse2, remind me whether I am right and please express your opinion here.

Reply:
I'll revert it. I got annoyed with the delay while loading/unloading it in Catalina, though I should've remembered to add it back. 👍